test/Python/python_runtime.swift

```diff
@@ -281,4 +307,38 @@ PythonRuntimeTestSuite.test("PythonRefCount") {
   }
 }
 
+PythonRuntimeTestSuite.test("ReferenceCounting") {
+  let referencedObjectCount = Int(Python.len(gc.get_objects()))!
```

Comment:
In `ReferenceCounting ` and `TraceMallocReferenceCounting`:  please perform operations/comparisons directly on `PythonObject` instead of casting to `Int`.

Is there a reason for calling `expectNotEqual(0, ...)` instead of `expectEqual`? It would be nice to call `expectEqual` with the exact expected value for extra confidence; "not equal to zero" is a weak condition.

It seems `let v = Python.list([1000, 2000, 3000])` causes `len(gc.get_objects())` to increase by 3, but in Python the same code causes an increase by only 1. Is this a refcounting bug that we should fix?

Reply:
No bug. Python only tracks in gc.get_objects() the objects that may be part of a cycle. You have to be careful in the python command line because repr([1,2,3]) produces an increment of 2. Thus,
you get something like:

```
Python ...
Type "help", "copyright", "credits" or "license" for more information.
>>> import gc
>>> len(gc.get_objects())
3845
>>> v = [1000,2000,3000]
>>> len(gc.get_objects())
3846
>>> print v
[1000, 2000, 3000]
>>> len(gc.get_objects())
3848
>>> 
```